src/use-dropdown-menu.ts

```diff
@@ -75,16 +78,19 @@ export default function useDropdownMenu(itemCount: number) {
 		if (isKeyboardEvent(e)) {
 			const { key } = e;
 
-			if (key !== 'Tab' && key !== 'Shift') {
-				e.preventDefault();
+			if (!['Enter', ' ', 'Tab', 'ArrowDown'].includes(key)) {
+				return;
 			}
 
-			if (key === 'Enter' || key === ' ') {
+			if ((key === 'Tab' || key === 'ArrowDown') && clickedOpen.current && isOpen) {
```

Comment:
This seems to remove the functionality of opening the menu to the first item with the space key.

Reply:
That functionality only applies when the menu is already open.